primary/src/block_synchronizer/mod.rs

```diff
@@ -57,6 +57,10 @@ pub mod responses;
 /// proceed to next state.
 const CERTIFICATE_RESPONSES_RATIO_THRESHOLD: f32 = 0.5;
 
+/// The maximum number of rounds to be included in each range request. Currently the same as the
+/// default GC threshold. We can make it into a parameter if needed.
+const CERTIFICATES_RANGE_REQUEST_MAX_ROUNDS: u32 = 50;
```

Comment:
I would actually set an internal value (not a constant) equal to precisely the `gc_depth` parameter.  I believe this will transition better than this magic constant (however well set).

Reply:
Good point!